FrontEnd/Hopsy/src/screens/HomeScreen.js

```diff
@@ -8,48 +8,11 @@ import Icon from "react-native-vector-icons/Ionicons";
 import { FlatList } from "react-native-gesture-handler";
 import moment from 'moment';
 
-posts = [{
-  id: "1",
-  name: "SLO Brew",
-  text:
-      "Polyvision concert this weekend!",
-  timestamp: Date.now(),
-  avatar: require("../images/Bunny.png"),
-  image: require("../images/Bunny.png")
-},
-{
-  id: "2",
-  name: "Bull's Tavern",
-  text:
-      "Bull sweat now 50% off!!!",
-  timestamp: Date.now(),
-  avatar: require("../images/Bunny.png"),
-  image: require("../images/Bunny.png")
-},
-{
-  id: "3",
-  name: "Eagle Brewing",
-  text:
-      "Eagle Grand Opening this weekend!",
-  timestamp: Date.now(),
-  avatar: require("../images/Bunny.png"),
-  image: require("../images/Bunny.png")
-},
-{
-  id: "4",
-  name: "Golden Coast Brewing",
-  text:
-      "25% off all beer this week only!",
-  timestamp: Date.now(),
-  avatar: require("../images/Bunny.png"),
-  image: require("../images/Bunny.png")
-}];
-
 export default class Test extends React.Component {
   renderPosts = post =>{
     return(
-      <View style={styles.feedItem}>
-        <Image source={post.avatar} style={styles.avatar}/>
+      <SafeAreaView style={styles.feedItem}>
+        <Image source={post.avatar} style={styles.avatar} resizeMode="contain"/>
```

Comment:
Maybe wrap this inside a component so it can be reused throughout the app.

Reply:
This does make sense but it is unlikely that this structure will be used elsewhere in the aop